.github/workflows/build-test-package.yml

```diff
@@ -172,6 +172,9 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install ninja
         python -m pip install itk==5.2rc2
+        python -m pip install itk-elastix
+        python -m pip install vtk
```

Comment:
`vtk` does not have Python 3.9 support, and we cannot add it as a dependency.

Reply:
Not sure I follow. This particular check `test-python-hasi-package` relies only on Python 3.7. If HASI Python scripts must support Python 3.6 - 3.9 does that mean that we cannot use the `vtk` Python package at all?

Reply:
Removed `vtk` package dependency.